### ISSUE TITLE
lost connection on python_tutor

### ISSUE THREAD

**Issue by geokai** [Koraavo/Hangman_new#1]
Sry, somehow lost the connection to the help session
hope I didn't confuse you too much with all that git/github stuff.
check out the git help pages

also good to check out:
Al Sweigart - https://automatetheboringstuff.com/ - really good beginner stuff!!
Dan Bader - Real Python @ https://realpython.com/ - very good python resource, loads of tuts

Hope it works out - happy coding : )

**Comment by Koraavo** [comment]
Hey thanks for writing me.
I have been looking into basic python stuff.
I believe the issue is not always the syntax but learning to problem solve
it
Once you have that sorted, the rest works...
By issue is still not resolved yet regarding the hangman thing.

But I will get back to it...
I hope you won't mind if I can sometimes write you if I am stuck :)

**Comment by geokai** [comment]
Indeed, problem solving is always the main challenge, in any subject.
Learning syntax and 'the rules' is the easier part.

My general python programing is limited (as you might see in my repos, mostly micropython for microcontrollers) but I'd be happy to offer any help I can.

I can certainly provide suggestions for sources of info - I've accumulated a huge list of on-line stuff:
git and github, there is very good help at github
.
Also youtube publishers such as Corey Schaffer, Derek Banas and others.

Dan Bader runs realpython.org - a fab learning resource

Al Sweigart wrote a book called 'Automate the Boring Stuff With Python'. It is available as a free on-line pdf.

If you have twitter follow Raymond Hettinger, he's one of the main Python Core Developers and his twitter mission is to teach (and help) people  through tweets.

Lots to explore - enjoy
